library/src/scripts/utility.ts

```diff
@@ -325,3 +325,13 @@ export function sanitizeUrl(url: string) {
         return "unsafe:" + url;
     }
 }
+
+/**
+ * Check if valid array
+ *
+ * @param items The array to validate
+ * @param minItems Minimum expected items, defaults to 1;
+ */
+export function loopableArray(items?: any[], minItems: number = 0) {
+    return !!items && items.length && items.length > minItems;
```

Comment:
Do we really need a utility for this? The intermediary items.length seems unnessary because we have what is ensured to be an array type which always has that property. In that case this utility function would allow us to use the following as equivalents:

```ts
if (items && items.length > 1) {}
// with
import {loopableArray} from "@library/utility";
if (loopableArray(items, 1) {}
```

It doesn't seem like a gain to me. If you still want to keep this function though I think it needs a better name because "loopable" doesn't mean much to me. A 0 length array is still loopable. You can iterate on it. Maybe something like `arrayHasMinLength()` but then you're longer in length than the original check.